src/mapping/NearestProjectionMapping.hpp

```diff
@@ -25,33 +25,11 @@ class NearestProjectionMapping : public Mapping {
   /// Computes the projections and interpolation relations.
   virtual void computeMapping() override;
 
-  virtual bool hasComputedMapping() const override;
-
-  /// Removes a computed mapping.
-  virtual void clear() override;
-
-  /**
-   * @brief Uses projection and interpolation relations to map data values.
-   *
-   * Preconditions:
-   * - computeMapping() has been called
-   *
-   * @param[in] inputDataID Data ID of input data values to be mapped from.
-   * @param[in] outputDataID Data ID of output data values to be mapped to.
-   */
-  virtual void map(
-      int inputDataID,
-      int outputDataID) override;
-
   virtual void tagMeshFirstRound() override;
   virtual void tagMeshSecondRound() override;
```

Comment:
It might be worth mentioning in some code comment, why the tagging has to be handled individually. So far, all mapping base classes share a tagging method in a common base class.

Reply:
Changed it while you were reviewing!